client/src/legacy/GridField.js

```diff
@@ -64,7 +64,13 @@ $.entwine('ss', function($) {
       // Include any GET parameters from the current URL, as the view state might depend on it.
       // For example, a list prefiltered through external search criteria might be passed to GridField.
       if(window.location.search) {
-        ajaxOpts.data = window.location.search.replace(/^\?/, '') + '&' + $.param(ajaxOpts.data);
+        let searchParams = window.location.search.replace('?', '').split('&');
+        for (let i = 0; i < searchParams.length; i++) {
+          let parts = searchParams[i].split('=');
+          if (parts.length == 2) {
+            ajaxOpts.data.push({name: parts[0], value: parts[1]});
```

Comment:
Don't these need url decoding or does `wiondow.location.search` come url decoded?

Reply:
`window.location.search` doesn't seem to be url encoded, at least not in Chrome of Firefox. I thought about using `URLSearchParams` for a bit cleaner code, but that is not supported in all legacy browsers (read IE11).

Reply:
I've done a little research and it looks like you do need to url decode the data: https://stackoverflow.com/a/2091331/4978044

Reply:
Ah, you are correct. Fixed in commit #2a3bf4c.